frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/state-update/vm-settings-tab-state-update.ts

```diff
@@ -176,7 +176,7 @@ const baseImageUpdater = ({ id, prevState, dispatch, getState }: UpdateOptions)
   if (!hasVMSettingsValueChanged(prevState, state, id, VMSettingsField.OPERATING_SYSTEM)) {
     return;
   }
-  if (iGetLoadedCommonData(state, id, VMWizardProps.isProviderImport)) {
```

Comment:
can we first ask if `isProviderImport` to be consistent?

Reply:
moved up :-)